plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -36,11 +36,18 @@
             GET /1.0/containers/<name>
             U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
             are different, they this module tries to apply the configurations.
-          - The key starts with 'volatile.' are ignored for this comparison.
```

Comment:
I would keep this, but add `when I(ignore_volatile_options=true)` at the end.

Reply:
done